### PR TITLE
Update app's level

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Review by jedie — APPROVED** [review]

File: apps.json [hunk 1]
```diff
@@ -494,7 +494,7 @@
     },
     "cryptpad": {
         "category": "office",
-        "level": 8,
+        "level": 2,
```
Comment (tituspijean) [inline apps.json:497]:
```suggestion
        "level": 8,
```

cf. last CI check: https://ci-apps.yunohost.org/ci/job/4479

File: apps.json [hunk 2]
```diff
@@ -3620,7 +3621,7 @@
     },
     "ztncui": {
         "category": "system_tools",
-        "level": 7,
+        "level": 0,
```
Comment (tituspijean) [inline apps.json:3624]:
```suggestion
        "level": 7,
```
cf. [Test Badge](https://ci.pijean.ovh/ci/job/67)

File: apps.json [hunk 3]
```diff
@@ -1866,8 +1866,8 @@
     },
     "minio": {
         "category": "dev",
-        "url": "https://github.com/yunohost-Apps/minio_ynh",
-        "state": "inprogress"
+        "state": "inprogress",
+        "url": "https://github.com/yunohost-Apps/minio_ynh"
```
Comment (tituspijean) [inline apps.json:1870]:
```suggestion
        "url": "https://github.com/YunoHost-Apps/minio_ynh"
```

**Comment by alexAubin** [comment]
- [ ] Emailpoubelle : https://ci-apps.yunohost.org/ci/job/4290
- [ ] Horde : https://ci-apps.yunohost.org/ci/job/4307

File: apps.json [hunk 4]
```diff
@@ -1237,7 +1237,7 @@
     },
     "horde": {
         "category": "communication",
-        "level": 6,
+        "level": 1,
```
Comment (tituspijean) [inline apps.json:1240]:
```suggestion
        "level": 6,
```

File: apps.json [hunk 5]
```diff
@@ -735,7 +735,7 @@
     },
     "emailpoubelle": {
         "category": "small_utilities",
-        "level": 6,
+        "level": 1,
```
Comment (tituspijean) [inline apps.json:738]:
```suggestion
        "level": 6,
```

**Review by tituspijean — APPROVED** [review]

**Pull request merged**